### ISSUE TITLE
Upgrade to React Native 0.39

### ISSUE THREAD

**Issue by prijindal** [GeekyAnts/NativeBase#354]
It seems like React native have done some breaking changes regarding layouts

https://github.com/facebook/react-native/wiki/Breaking-Changes#move-to-new-c-based-implementation-of-css-layout-on-android-d63ba4---astreet

> Previously, developers had to put flex: 1 in many places it didn't belong in order to work around a bug in css-layout. Now flex: 1 is treated properly and, unfortunately, this means that your layout may no longer look correct. Specifically, you may see that your layout looks collapsed, or children don't render. The fix is to simply remove flex: 1 from those containers.

As native base is using flex: 1 in a lot of places, won't that mean there would be a lot of breaking changes?

**Comment by trevorpreston** [comment]
This explains a lot.  I've recently run into an issue where wrapping my code in a <Container> component makes the children disappear.

**Comment by richard-sim** [comment]
Yep, this explains a lot alright. I'm currently trying to evaluate NativeBase, so I've been trying out a bunch of tutorials/examples in new test projects - all of which are horribly broken. Not NB's fault obviously, but wow it needs to be fixed.

**Comment by sanketsahusoft** [comment]
@richard-sim Expect it to be fixed in next couple of days for sure! Thanks for checking out NativeBase. :-)

**Comment by sankhadeeproy007** [comment]
We've upgraded [KitchenSink](https://github.com/GeekyAnts/NativeBase-KitchenSink) to the latest react-native and all the components seem to work pretty well. 
Any specific use cases you guys have which might have broken?
Would be nice if we knew where actually it is breaking.

**Comment by richard-sim** [comment]
Sorry for being vague! The nativebase-tutorial repo demonstrates it well; all the Text components from the Card that opens when you tap an item in the list are missing, leaving only the items image, name, and the Go Back button visible in the card.

I've forked it and updated all the dependencies to their latest versions to make it easy to verify the issue: https://github.com/richard-sim/nativebase-tutorial

**Comment by sankhadeeproy007** [comment]
Thanks for getting back! We'll look into this soon :)

**Comment by sankhadeeproy007** [comment]
@richard-sim It was a bad idea to use cards for that in the first place. As we had to modify that a lot. Would have been better off with a plain View. Also I don't think it has anything to do with react-native upgrade to 0.39. We've changed the render logic inside CardItem since then hence that was a breaking change in `native-base` itself. A better example would be to check the KitchenSink app where an update to 0.39 doesn't break anything.
